profile.md

```diff
@@ -658,11 +658,17 @@ If an entity is not entitled to a capability, the scope requested may be ignored
 
 ## Group or Role Based Capability Selection
 
-An entity may be entitled to a capability due to membership in a group or entitlement to use a role. The entity may be a member of multiple groups (VOs), with multiple roles, supported by a common implementation (analogous to how VOMS-Admin is operated at CERN). To support this scenario, a `wlcg.capabilityset` value may be included in the scope request to specify the group/role context for the scope request. This can determine the resulting `iss` and `scope` claims in the issued token.
+An entity may be entitled to capabilities due to membership in a group or entitlement to use a role. The entity may be a member of multiple groups (VOs), with multiple roles, supported by a common implementation. In addition, a client shared by multiple entities may not know which capabilities are available to each entity, and the token issuer has that knowledge.
+
+To support this scenario, a `wlcg.capabilityset` scope MAY be included in the scope request to specify the group/role context.  The parameter given with the `wlcg.capabilityset` scope is exactly the same as the `group` used with `wlcg.groups` as specified in the [Common-Claims section](#Common-Claims) above.  This can determine the resulting `scope` claims in the issued token.
```

Comment:
The WLCG JWT profile does not define role contexts. I would remove the /role context in the sentence above